src/resources/postcss/tribe-common-admin.pcss

```diff
@@ -2118,6 +2133,77 @@ input[type="email"].tribe-events-admin-card__input {
 	}
 }
 
+.tribe-events-admin-notice {
+	background-color: #3D54FF;
+	height: 65px;
+	padding-top: 13px;
+
+	.tribe-events-admin-content-wrapper {
+		padding-top: 0;
+		padding-bottom: 0;
+	}
+
+	p {
+		color: #fff;
+		display: inline-block;
+		font-family: Helvetica;
+		font-size: 16px;
+		line-height: 18px;
+		margin-top: 0px;
+		padding-bottom: 12px;
+		padding-left: 16px;
+		vertical-align: middle;
+	}
+}
+
+.tribe-events-admin-notice__logo {
+	display: inline-block;
+}
+
+/* -- Event Tickets tweaks -- */
+.tribe-events-admin-tickets {
```

Comment:
I wonder if because these are specific to Ticket it should live in ET files.

Reply:
I don't think so, all of this is in Common itself, not in ET